workers/unity/Packages/com.improbable.gdk.transformsynchronization/ScriptableObjects/TransformSynchronizationReceiveStrategy.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.TransformSynchronization
+{
+    public abstract class TransformSynchronizationReceiveStrategy : ScriptableObject
```

Comment:
The abstract classes for receive and send are the same - is it worth folding them into one and the same?

Reply:
No, we can't detect if the user has used it correctly otherwise.